cocktails9/cocktails9/API/CocktailService.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class CocktailService {
+    
+    private let networkManager: NetworkManager
+    private let baseURL = "https://www.thecocktaildb.com/api/json/v1/1/filter.php?a=Alcoholic"
```

Comment:
baseUrl is usually what it says. base url. Then, you can add paths to it later on if needed
```suggestion
    private let baseURL = "https://www.thecocktaildb.com/api/json/v1/1/"
```